titus_isolate/isolate/cpu.py

```diff
@@ -1,11 +1,124 @@
+from collections import defaultdict
+
 from titus_isolate.model.processor.utils import get_emptiest_core, is_cpu_full
 from titus_isolate.model.workload import Workload
 from titus_isolate.utils import get_logger
 
+from titus_optimize.compute import optimize_ip
+
 log = get_logger()
 
+def assign_threads(cpu, workload, workload_insertion_times={}):
```

Comment:
Can you please document the `workload_insertion_times` parameter.  It's unclear to me what it's role is.

Reply:
done